insteonplm/plm.py

```diff
@@ -343,7 +342,7 @@ def _setup_devices(self):
                       len(self.devices.saved_devices))
         self._get_plm_info()
         self.devices.add_known_devices(self)
-        if self._quick_start:
+        if self._load_aldb == False:
```

Comment:
Change the order of this: 
If self._load_aldb:
    self._load_all_link_database()
else:
    self._complete_setup()